external-crates/move/move-compiler/tests/move_check/typing/continue_any_type.exp

```diff
@@ -6,6 +6,18 @@ warning[W09005]: dead or unreachable code
   │
   = This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
+warning[W09004]: unnecessary trailing semicolon
+  ┌─ tests/move_check/typing/continue_any_type.move:6:25
+  │
+6 │             0 + continue;
```

Comment:
(I am reviewing these in reverse order, does GitHub list out my comments in file order or in comment order?)
But we should not emit this warning here, if you remove the `;` it will fail to type check